src/runtime/node/querystring/index.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  parseQuery,
+  stringifyQuery,
+  encode as _encode,
+  decode as _decode,
+} from "ufo";
+import type { QueryObject } from "ufo";
+import type querystring from "node:querystring";
+import { notImplemented } from "src/runtime/_internal/utils";
+
+const _stringify: typeof querystring.stringify = function (obj) {
+  return stringifyQuery(obj as QueryObject);
+};
+export const decode: typeof querystring.decode = parseQuery;
```

Comment:
While we don't respect/support all params (sep and eq), it might be more clear that we create a wrapper function (this way is also more runtime safe, the second arg `sep` won't be accidentally passed as second arg of `parseQuery`)